hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/store/contracts/AbstractCodeCache.java

```diff
@@ -50,14 +51,16 @@ public Code getIfPresent(final Address address) {
 
         if (entityAccess.isTokenAccount(address)) {
             final var interpolatedBytecode = proxyBytecodeFor(address);
-            code = Code.createLegacyCode(interpolatedBytecode, Hash.hash(interpolatedBytecode));
+            code =
+                    CodeFactory.createCode(
+                            interpolatedBytecode, Hash.hash(interpolatedBytecode), 0, false);
             cache.put(cacheKey, code);
             return code;
         }
 
         final var bytecode = entityAccess.fetchCodeIfPresent(address);
         if (bytecode != null) {
-            code = Code.createLegacyCode(bytecode, Hash.hash(bytecode));
+            code = CodeFactory.createCode(bytecode, Hash.hash(bytecode), 0, false);
```

Comment:
The added `0, false` is not self-explanatory. Since this change is repeated in many files, maybe we should use a wrapper function and/or use constants to remove the magic values.

Reply:
I agree with you, but this isn't the place to fix that. The change you're looking at was part of PR 4401 which was cherry-picked from develop into this release branch. Really it should be fixed in develop branch.

Reply:
PR 4401 has already been merged into develop. I will leave it to you whether to open an issue to fix this in the future.

Reply:
As I mentioned, this is not a blocker. It just means there are rooms for improvement (on readability in this case)

Reply:
I created #4409 and assign @shemnon  and yourself @kimbor. Thank you!